### PR TITLE
Fix getSizeInBytes for VariableWidth- and Interleaved-Block

### PR DESCRIPTION
**Pull request by haozhun**

### CODE REVIEW

File: presto-main/src/test/java/com/facebook/presto/block/TestInterleavedBlock.java [hunk 1]
```diff
@@ -61,6 +66,37 @@ public void testCopyPositions()
...
+        Block quarter2 = half1.getRegion(numEntries / 4, numEntries / 4);
+        Block quarter3 = half2.getRegion(0, numEntries / 4);
+        Block quarter4 = half2.getRegion(numEntries / 4, numEntries / 4);
+
+        int sizeInBytes = block.getSizeInBytes();
+        int quarter1size = quarter1.getSizeInBytes();
+        int quarter2size = quarter2.getSizeInBytes();
+        int quarter3size = quarter3.getSizeInBytes();
+        int quarter4size = quarter4.getSizeInBytes();
+        assertTrue(quarter1size > sizeInBytes * 0.2 && quarter1size < sizeInBytes * 0.3, format("quarter1size is %s, should be between %s and %s", quarter1size, sizeInBytes * 0.2, sizeInBytes * 0.3));
```
Comment (dain) [inline presto-main/src/test/java/com/facebook/presto/block/TestInterleavedBlock.java]: add variables `expectedQuarterSizeMin` and expectedQuarterSizeMax`

File: presto-main/src/test/java/com/facebook/presto/block/TestVariableWidthBlock.java [hunk 2]
```diff
@@ -56,6 +62,36 @@ public void testCopyPositions()
...
+        Block quarter2 = half1.getRegion(numEntries / 4, numEntries / 4);
+        Block quarter3 = half2.getRegion(0, numEntries / 4);
+        Block quarter4 = half2.getRegion(numEntries / 4, numEntries / 4);
+
+        int sizeInBytes = block.getSizeInBytes();
+        int quarter1size = quarter1.getSizeInBytes();
+        int quarter2size = quarter2.getSizeInBytes();
+        int quarter3size = quarter3.getSizeInBytes();
+        int quarter4size = quarter4.getSizeInBytes();
+        assertTrue(quarter1size > sizeInBytes * 0.2 && quarter1size < sizeInBytes * 0.3, format("quarter1size is %s, should be between %s and %s", quarter1size, sizeInBytes * 0.2, sizeInBytes * 0.3));
```
Comment (dain) [inline presto-main/src/test/java/com/facebook/presto/block/TestVariableWidthBlock.java]: Same here

File: presto-docs/src/main/sphinx/release/release-0.152.rst [hunk 3]
```diff
@@ -17,6 +17,7 @@ General Changes
 * Display port on the task list in the web UI when multiple workers share the same host.
 * Add support for ``EXCEPT``.
 * Rename ``FLOAT`` type to ``REAL`` for better compatibility with the SQL standard.
+* Fix potential performance bug when transporting rows over network
```
Comment (dain) [inline presto-docs/src/main/sphinx/release/release-0.152.rst]: `Fix potential performance regression when transporting rows containing an array, map or row type between nodes`

Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.152.rst]: End release note items in a period.

**Pull request merged**